crds/jwst/specs/niriss_pars-chargemigrationstep.rmap

```diff
@@ -0,0 +1,17 @@
+header = {
+    'derived_from' : 'Derived from niriss_pars-jumpstep 2023-09-07',
+    'file_ext' : '.asdf',
+    'filekind' : 'pars-chargemigrationstep',
+    'filetype' : 'pars-chargemigrationstep',
+    'instrument' : 'NIRISS',
+    'mapping' : 'REFERENCE',
+    'name' : 'jwst_niriss_pars-chargemigrationstep.rmap',
+    'observatory' : 'JWST',
+    'parkey' : (('META.INSTRUMENT.EXP_TYPE', 'META.INSTRUMENT.FILTER', 'META.INSTRUMENT.PUPIL'),),
```

Comment:
Ok, I wasn't certain it needed it since we have an rmap like miri_v2v3.rmap which has:
'parkey' : (('META.INSTRUMENT.DETECTOR', 'META.INSTRUMENT.EXP_TYPE', 'META.INSTRUMENT.CHANNEL', 'META.INSTRUMENT.BAND'),),

but I will get that second part added.